contrib/db_pools/lib/src/pool.rs

```diff
@@ -254,7 +254,6 @@ mod mongodb {
             opts.min_pool_size = config.min_connections;
             opts.max_pool_size = Some(config.max_connections as u32);
             opts.max_idle_time = config.idle_timeout.map(Duration::from_secs);
-            opts.wait_queue_timeout = Some(Duration::from_secs(config.connect_timeout));
```

Comment:
This option was removed in the `2.0.0` release; see [RUST-757](https://jira.mongodb.org/browse/RUST-757) for more information.

Reply:
The suggested client-side change appears to be to wrap the affected operations with a timeout manually. Which operations honored the `wait_queue_timeout`, and are any of those already covered by `connect_timeout`?